src/bitcoin.js

```diff
@@ -203,7 +192,22 @@ exports.serializeTx = function(data) {
 
 // Convert a pubkeyhash to a bitcoin base58check address with a version byte
 exports.getBitcoinAddress = function(pubkeyhash, version) {
-  return bs58check.encode(Buffer.concat([Buffer.from([version]), pubkeyhash]));
+  let bech32Prefix = null;
+  let bech32Version = null;
+  if (version === FMT_SEGWIT_NATIVE_V0) {
+    bech32Prefix = SEGWIT_NATIVE_V0_PREFIX;
+    bech32Version = SEGWIT_V0;
+  } else if (version === FMT_SEGWIT_NATIVE_V0_TESTNET) {
+    bech32Prefix = SEGWIT_NATIVE_V0_TESTNET_PREFIX
+    bech32Version = SEGWIT_V0;
+  }
+  if (bech32Prefix && bech32Version !== null) {
```

Comment:
Would be better to use `bech32Prefix !== null`, no?

Reply:
5824a88

Reply:
5824a88